src/mmw/js/src/core/views.js

```diff
@@ -59,6 +61,10 @@ var HeaderView = Marionette.ItemView.extend({
         // Defer requiring app until needed as it is not defined when
         // core.views are initialized (they are required in app.js)
         require('../app').showLoginModal();
+    },
+
+    userLogout: function() {
+        require('../app').userLogout();
```

Comment:
Maybe this is a good case for Backbone.Wreqr which I think is already part of Marionette. We might be able to handle this through the radio feature:
https://github.com/marionettejs/backbone.wreqr